packages/uikit-react-native-foundation/src/context/CustomComponentCtx.tsx

```diff
@@ -63,6 +63,12 @@ export type SendInputRenderProp = (props: {
 
 export type CustomProvidersRenderProp = ({ children }: { children: React.ReactElement }) => React.ReactElement;
 
+export type CombinedNewMessagesScrollToBottomButtonRenderProp = (props: {
```

Comment:
you might also want to export this type from the pkg index.ts file if you want to use it on the other end instead of defining it again. Although I guess there's an argument for just defining it again